Tools/Scripts/hooks/prepare-commit-msg

```diff
@@ -11,26 +12,39 @@ SCRIPTS = os.path.dirname(os.path.dirname(LOCATION))
 CHERRY_PICKING_RE = re.compile(r'^# You are currently cherry-picking commit (?P<hash>[a-f0-9A-F]+)\.$')
 IDENTIFIER_RE = re.compile(r'(\d+\.)?\d+@\S+')
 PREFER_RADAR = {{ prefer_radar }}
+ENVIRONMENT = os.environ.copy()
 
 sys.path.append(SCRIPTS)
 from webkitpy.common.checkout.diff_parser import DiffParser
 from webkitbugspy import radar
 
 
+cwd = ENVIRONMENT.get('PWD', os.getcwd())
+environment_file = os.path.join(cwd, '.git', 'commit-message-environment.json')
```

Comment:
Given this is a hook, shouldn't we use `ENVIRONMENT['GIT_DIR']` here?

Reply:
https://git-scm.com/docs/githooks certainly claims that in a hook:

> Environment variables, such as GIT_DIR, GIT_WORK_TREE, etc., are exported so that Git commands run by the hook can correctly locate the repository.

Reply:
Maybe they are only exported if they are set? Don't see those when dumping the environment in our `prepare-commit-msg` hook.

Reply:
We don't need any of this with `git rev-parse --absolute-git-dir`, `git` will figure out everything from there, it would seem.